WordPress/Classes/Extensions/UIAlertController+Helpers.swift

```diff
@@ -16,7 +16,42 @@ extension UIAlertController {
         while leafViewController.presentedViewController != nil && !leafViewController.presentedViewController!.isBeingDismissed {
             leafViewController = leafViewController.presentedViewController!
         }
-
         leafViewController.present(self, animated: true, completion: nil)
     }
 }
+
+//MARK: - copy text to Clipboard
+extension UIAlertController {
+    /// This method is used for presenting the Action sheet
+    /// for copying text to clipboard. The action sheet has 2 options:
+    /// copy: will copy the text to the clipboard
+    /// cancel: dismiss the action sheet
+    @objc static func copyTextAlertController(_ text: String?,
+                                              completion: ((Bool) -> Void)? = nil) -> UIAlertController? {
+        guard let text = text else {
+            completion?(false)
+            return nil
+        }
+        let alertController = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        alertController.addDefaultActionWithTitle(NSLocalizedString("Copy", comment: "Copy button")) { _ in
```

Comment:
Two nitpicks here: "Copy" is an inaccurate description of what the button _actually does_: it copies the comments _URL_, not itself.

The comment is also a bit lacking — remember that translators who will only see the context we provide!  Briefly explaining _when_ that button shows would be cool to see here :)

Reply:
👍 makes sense. Updating to "Copy Link to Comment"!